### PR TITLE
chore: openfin/core upgrade and prettier formatting

### PR DESCRIPTION
**Pull request by SHolleworth**

### CODE REVIEW

File: package-lock.json
Comment (algreasley) [inline package-lock.json:1]: remove this file, please, it doesn't belong here in a yarn world (prob sync with RT and go npm in future)

File: client/src/ra-platforms/fdc3/FDC3Context.tsx [hunk 1]
```diff
@@ -16,13 +16,13 @@ export const FDC3Provider: React.FC = ({ children }) => {
...
     const setContext = (context: Context) => {
-      if (context.type === 'fdc3.instrument' && context.id?.ticker) {
+      if (context.type === "fdc3.instrument" && context.id?.ticker) {
         setCurrentSymbol(context.id?.ticker)
       }
     }
 
     if (window.fdc3) {
-      joinChannel('green').then(() => {
+      joinChannel("green").then(() => {
```
Comment (algreasley) [inline client/src/ra-platforms/fdc3/FDC3Context.tsx:25]: note (to self, mainly 😄) .. why the hardcoded "green" .. ref Moise's findings over on Finsemble ..

File: client/package.json [hunk 2]
```diff
@@ -27,7 +30,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.8.1",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.2",
-    "@openfin/core": "^29.x",
+    "@openfin/core": "31.75.4",
     "downshift": "^4.0.7",
     "graphql": "16.6.0",
     "lodash": "^4.17.19",
```
Comment (algreasley) [inline client/package.json:36]: "while you're here" .. please update to specific version (.21) ... lodash is a serial gift-giver of vulnerabilities ..

File: client/package.json [hunk 3]
```diff
@@ -65,8 +68,9 @@
     "cross-env": "^7.0.2",
     "happy-dom": "^8.1.0",
```
Comment (algreasley) [inline client/package.json:69]: "while you're here" .. remove "happy-dom", we don't use it

File: client/package.json [hunk 4]
```diff
@@ -65,8 +68,9 @@
...
     "jsdom": "^20.0.3",
-    "openfin-cli": "^3.0.2",
+    "openfin-cli": "^4.0.0",
     "polished": "^3.2.0",
+    "prettier": "2.8.8",
     "typescript": "^4.9.3",
```
Comment (algreasley) [inline client/package.json:74]: optional, while "upsetting the apple cart" .. typescript / vite updates <=> RT ?

File: client/src/common/StyledComponents.tsx [hunk 5]
```diff
@@ -1,9 +1,9 @@
-import styled, { css } from 'styled-components/macro'
+import styled, { css } from "styled-components/macro"
 
-import { default as dataCard } from './DataCard'
-import { fonts } from '@/rt-theme/fonts'
-import { pxToRems } from '@/utils'
-import { mediaQuery } from '@/rt-theme/mediaQueries'
+import { default as dataCard } from "./DataCard"
```
Comment (algreasley) [inline client/src/common/StyledComponents.tsx:3]: reminds me, we need to apply the same linting guff to this repo (another RA <=> RT task .. later, I think, unless you have thumb-twiddling time .. it's a turn-everything upside down kind of task, so now as good a time as any .. perhaps a separate PR is appropriate tho, eh?

**Review by algreasley — COMMENTED** [review]
I think you are missing the actual version update in the manifest 😄

File: client/package.json (hunk 3 above)
Comment (algreasley) [reply client/package.json:69]: ignore .. I have done this in my branch

File: client/package.json (hunk 4 above)
Comment (algreasley) [reply client/package.json:74]: ignore .. I have done this in my branch

File: client/package.json (hunk 2 above)
Comment (algreasley) [reply client/package.json:36]: ignore .. I have done this in my branch

File: client/src/containers/search/graphql/types/search.ts [hunk 6]
```diff
@@ -3,24 +3,24 @@
 // @generated
 // This file was automatically generated and should not be edited.
```
Comment (algreasley) [inline client/src/containers/search/graphql/types/search.ts:4]: hmm .. auto-generated .. should we be auto-generating the updated version?

Comment (algreasley) [reply client/src/containers/search/graphql/types/search.ts:4]: probs just a normal prettier-on-save thing ..

File: server/src/graph-ql/DateScalars.ts [hunk 7]
```diff
@@ -1,12 +1,12 @@
-import { GraphQLScalarType, Kind } from 'graphql'
-import moment from 'moment'
+import { GraphQLScalarType, Kind } from "graphql"
+import moment from "moment"
```
Comment (algreasley) [inline server/src/graph-ql/DateScalars.ts:2]: another note to our future selves .. the world has moved on from moment, so we should perhaps move with it in RA ..

File: client/src/common/StyledComponents.tsx (hunk 5 above)
Comment (SHolleworth) [reply client/src/common/StyledComponents.tsx:3]: what is the linting guff?

Comment (algreasley) [reply client/src/common/StyledComponents.tsx:3]: stuff like the import order and such (per Ignacio's PR for lint etc. a few months ago on RT)

**Review by algreasley — APPROVED** [review]
great .. let's circle back next time we are doing RA admin to sync up the vite config a bit more with RT .. <..> tags for all substitions .. and I saw some imports that could perhaps do with aliasing, but no point in doing anything major there as we are considering absorbing RA into RT repo 'workspace'

**Pull request merged**